sdk/apimanagement/api-management-custom-widgets-scaffolder/package.json

```diff
@@ -96,10 +96,12 @@
     "./package.json": "./package.json",
     ".": {
       "import": {
+        "source": "./src/index.ts",
```

Comment:
I think this is new to tshy - @mpodwysocki / @jeremymeng do you think we should run `rush build` on `main` and commit the `source` addition so PRs do not get them accidentally?

Reply:
yes I think so

Reply:
Ok, I can open a PR with this shortly. I'm running against all 391 packages but when that's done I only expect a few files (the ones in this PR, really).